docker/dev/Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM python:3.5
+
+WORKDIR /usr/src/app
+
+ENV APP_NAME respa
+
+RUN apt-get update && apt-get install -y libgdal1h postgresql-client-9.4
+
+COPY requirements.txt ./requirements.txt
+COPY dev-requirements.txt ./dev-requirements.txt
+
+RUN pip install --no-cache-dir -r dev-requirements.txt
+
```

Comment:
Cannot see where you are installing the regular `requirements.txt` 🤔

Reply:
See [this](https://github.com/City-of-Helsinki/respa/blob/master/dev-requirements.txt#L1).